server/src/test/java/org/cloudfoundry/identity/uaa/util/UaaUrlUtilsTest.java

```diff
@@ -78,7 +80,9 @@
         "http://sub*.valid.com/*/with/path**",
         "http://*.domain.com",
         "http://username:password@some.server.com",
-        "http://username:password@some.server.com/path"
+        "http://username:password@some.server.com/path",
```

Comment:
can we please add use cases: http://localhost:*/** and http://localhost:*/*

Reply:
Added!